### ISSUE TITLE
Problem with content-type 

### ISSUE THREAD

**Issue by robincsamuel** [mrin9/RapiDoc#192]
HI,

I'm facing an issue with parsing the request body. It looks like there is an extra semicolon in the content type (image attached).

I'm using `body-parser` with express.js and [here is a link](https://github.com/expressjs/body-parser/issues/264) explaining the same issue.

<img width="425" alt="Screenshot 2020-04-24 at 7 38 01 PM" src="https://user-images.githubusercontent.com/4545992/80221793-699cf600-8663-11ea-9631-b80d4bc01564.png">

Please let me know if you need any other details. 

Btw, thanks for this wonderful component :)

**Comment by mrin9** [comment]
This is already fixed in 6337f0c2457b1615fde4597f37a0e79f8c9fe0ff commit.

However this is not yet released. You need to build it at your end with the latest source

after git clone
```shell
 yarn install
 yarn build  # generates rapidoc-min.js in ./dist folder
 # yarn build_windows  (if you are on windows)
```
to test it before building use `yarn serve`

Let me know if that worked out on your end

**Comment by robincsamuel** [comment]
Great, thanks for the update.
